src/create-glamorous.js

```diff
@@ -109,7 +112,7 @@ function createGlamorous(splitProps) {
             forwardProps: fp,
             ...options,
           },
-        )(GlamorousComponent.styles)
```

Comment:
Turns out we didn't need this at all, and it's actually more performant to not have this because we'd get duplicate styles :scream: